rpi_backlight.py

```diff
@@ -155,21 +202,23 @@ def cli() -> None:
         for arg in (
             args.off,
             args.on,
+            args.toggle,
             args.brightness,
             args.max_brightness,
             args.actual_brightness,
             args.power,
+            args.duration, 
```

Comment:
I'll have to look into this again, pretty sure it worked before

Reply:
Didnt work with -b 0